library/validation/validate_core.php

```diff
@@ -0,0 +1,47 @@
+<?php
```

Comment:
Need a standard header. Check out here for a good example by your group:
https://github.com/matrix-israel/openemr/blob/131941db31d1ccff03ecc3778da9dbd1515135ab/library/validation/LBF_Validation.php#L2-L20
(also ensure place a copyright at the top of the header as this example(but use your or your company's info and not Rod's):
https://github.com/matrix-israel/openemr/blob/131941db31d1ccff03ecc3778da9dbd1515135ab/interface/patient_file/summary/add_edit_issue.php#L5
)